src/Chirp.Razor/Pages/Public.cshtml.cs

```diff
@@ -7,6 +7,9 @@ public class PublicModel : PageModel
 {
     private readonly ICheepService _service;
     public List<CheepDto> Cheeps { get; set; } = new List<CheepDto>();
+    [BindProperty]
+    public string Message { get; set; }
```

Comment:
Non-nullable property 'Message' must contain a non-null value when exiting constructor. Consider adding the 'required' modifier or declaring the property as nullable.